picard/tagger.py

```diff
@@ -412,7 +412,36 @@ def process(result=None, error=None):
                     self.add_files(result)
                 thread.run_task(get_files, process)
 
-        process(True, False)
+        def get_files_non_recursive():
+            listings = [f for f in os.listdir(path)]
+            if ignore_hidden:
+                listings = filter(lambda x: not is_hidden(x), listings)
+            listings = map(lambda x: os.path.join(path, x), listings)
+            files = filter(lambda x: os.path.isfile(x), listings)
+            number_of_files = len(files)
+            if number_of_files:
+                mparms = {
+                    'count': number_of_files,
+                    'directory': path,
+                }
+                log.debug("Adding %(count)d files from '%(directory)s'" %
+                          mparms)
+                self.window.set_statusbar_message(
+                    ungettext(
+                        "Adding %(count)d file from '%(directory)s' ...",
+                        "Adding %(count)d files from '%(directory)s' ...",
+                        number_of_files),
+                    mparms,
+                    translate=None,
+                    echo=None
```

Comment:
I was replicating the existing behaviour for get_files, should that be changed too?